app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -89,3 +89,5 @@ CONFIG_DEBUG_COREDUMP_MEMORY_DUMP_MIN=y
 
 CONFIG_PROBE=y
 CONFIG_PROBE_DMA_MAX=2
+
+CONFIG_CROSS_CORE_STREAM=y
```

Comment:
Lets also add this for all IPC4 targets including CAVS 2.5 (i.e. get more test coverage in CI).